### PR TITLE
Add non bucketed call duration attribute

### PR DESCRIPTION
**Pull request by typfel**
https://wearezeta.atlassian.net/browse/ZIOS-7932

### CODE REVIEW

File: Wire-iOS/Sources/Analytics/Events/Analytics+CallEvents.m [hunk 1]
```diff
@@ -149,7 +149,8 @@ - (NSDictionary *)attributesForCallDuration:(NSTimeInterval)duration
         durationSlot = @"[1h-infinite]";
     }
     
-    return @{ @"duration" : durationSlot };
+    return @{ @"duration" : durationSlot,
+              @"durationActual" : @(ceil(duration))};
```
Comment (marcoconti83) [inline Wire-iOS/Sources/Analytics/Events/Analytics+CallEvents.m]: what's the unit of measure? will this approximate to seconds?

Comment (typfel) [inline Wire-iOS/Sources/Analytics/Events/Analytics+CallEvents.m]: unit of measure is seconds without fractions

Comment (typfel) [inline Wire-iOS/Sources/Analytics/Events/Analytics+CallEvents.m]: `duration` is NSTimeInterval which is measured in seconds.

**Comment by typfel** [comment]
@marcoconti83  what change are you requesting?

**Pull request merged**